controllers/controllers.js

```diff
@@ -51,8 +56,40 @@ const getArticles = (req, res, next) => {
     })
 }
 
+const patchArticleVotes = (req, res, next) => {
+    const {article_id} = req.params
+    const {inc_votes} = req.body
 
+    if(!inc_votes || typeof inc_votes !== 'number' || Object.keys(req.body).length > 1){
+        return res.status(400).send({ msg: 'invalid post object'})
```

Comment:
This is not a Promise reject, keep your error handling in your model and you can then check your data structure and reject accordingly.